### DISCUSSION TITLE
Government search & seizures, state survaillance

### DISCUSSION THREAD

**Discussion by stevenkain** [liberland/constitution#592] [General]
### **The issue:**

The current Constitution gives power to the government to **enact search and seizures**, as well as **mass surveillance** of the citizens based on warrants issues by the Judiciary branch (of the govt.) upon **reasonable** motive - **[Art. 71](https://github.com/liberland/Constitution/blob/master/Constitution.md)**

### **Argument for removal from the Constitution:**

- This is not a limitation of State powers but an legislative back-door for the state to conduct such actions based on a piece of paper (warrant) issued by just another branch of the same govt.
- Nowhere is it defined what constitutes reasonable motive to enact such violation of citizen rights. (legislative ambiguity)
- It  does not fall within the role of minarchist govt. to conduct such coercive actions in Libertarian society 
- Enacting such actions will require a constant police force and/or state military, which Liberland does not have, nor a way to finance perpetually. It can only be achieved by expanding the govt. with and introducing a forced taxation system, thous moving further away from any Libertarian principles.
- Outsourcing the forceful violation of citizen rights to either a voluntary group of citizens or worse, an outside organization will de facto give these vigilantes/non-Liberlanders greater powers and privileges than Liberlanders have constitutionally, thous delegating the citizen's powers greater than they (don't) have. **No institution/organization can delegate collective powers that none of its members have, individually.**
- Neither the [**Swiss**](https://www.fedlex.admin.ch/eli/cc/1999/404/en#art_13) nor the [**Singaporean constitution**](https://sso.agc.gov.sg/Act/CONS1963?WholeDoc=1) have such exceptions inserted or delegate the power to the government to conduct such suspensions of citizen's rights. So why does the Liberland Constitution has to have them, considering it holds itself MORE libertarian than both of the above?

### **Better solutions for preventing crime:**

**Best solution:** Let the private market 100% handle it.

**Middle ground:** create an incentive-based, balanced govt. system of endorsement of fair business conduct as well as de-platforming/dis-incentivizing bad players, such as revoking a company's listing in the Liberland Business Registry (or an individual's e-residency) upon several (3 ?) private court verdicts of misconduct, unwillingness to disclose necessary information to private investigators/judges when a crime has been committed. 
**Mention:** the individual/organization in question must always be allowed a certain time to correct/remunerate its misconduct, if possible and accepted by the party victimized. (via private mediation)

**Extreme cases:** the issue of loss of citizenship/status should also be considered for extreme cases, such as repeated sedition against Libertarian principles, violation of the N.A.P. (including abuse of property rights)

The initial idea of the **Merit system as a collateral** can also be applied, if not for all but most minor misconduct, via an insurance requirement paid at the signing of any contract. This would also fall under the "Let the private market handle it".

**Comment by michalptacnik** [comment]
Good morning and thank you for your remark. I have just committed a first attempt to answer this recommendation. I realize that "shall be free from unreasonable searches" and "except" are two cumulative exceptions, creating a possible room for abuse.
I have, therefore, removed the except part from Art. 71. Now it states:
Art. 71: Right to Privacy

§ 1

The Government shall not violate the right of the people to be secure in their persons, houses, papers, and effects against unreasonable searches and seizures.
The same shall apply to any kind of surveillance of communication or activities. The Government shall never contract a private party or another government to perform such activities.
§ 2

The Judiciary shall only issue Warrants of search, seizure, or surveillance upon probable cause, particularly describing the place to be searched and the persons or things to be seized, as well as the reason for the search or seizure.

Hence:
- the only searches and seizures, as well as breaches of security of persons, houses, papers and effects are the _reasonable_ ones.
- what is reasonable is up to the judiciary, which, mind you, can be private.
- Government shall not do mass surveillance of any kind and shall not abuse public private partnerships in order to do it.
- nothing in this paragraph prevents private judiciaries or private investigations
- private judiciary in criminal matters is, however, prohibited elsewhere in this Constitution, and for good reasons - Scientology courts where n-theta evidence is impermissible and "SPs" are auto-guilty or Sharia courts where women can get stoned for being raped come to mind. We can't allow for this to be even remotely possible, the international community would laugh at us and it is immoral.
- For the same reason, I strongly advocate against adopting the American First Ammendment, which allows you to sell yourself to slavery in the name of religions, at least with respect to removing religious *civil* liability for matters like abuse.

**Comment by stevenkain** [comment]
**§ 1** still implies that "**reasonable**" search and seizures are powers given (delegated) to the agents of the government, by the citizens; **powers that the citizens DO NOT HAVE** hence delegate to the govt. agents.

**§ 2** still empowers the Judiciary (govt. branch) to issue Warrants with the above goal.

Hence, **nothing changed** de facto, only in form.
None of my above mentioned arguments or solutions have been addressed.

### Addressing further points raised in "Hence":

- **a)** Exactly: once again, the main argument is that there is **no reasonable motive** for the govt. or its agents to have such **unmerited** and **unjustified powers**, nor to violate the NAP of its citizens. Further more, even if by citizen vote such an anti-libertarian decree would be decided to be made law, there is **no reason** for these powers to be ratified **within the Constitution**, as I previously argued.

- **b)** if "the Judiciary" is private, and therefore not an official branch of the government, there is no reason for it to be given governmental powers within the Constitution (nor to be called "the Judiciary" for that matter).

- **c)** that is not what **§ 1** implies. In fact, it very clearly states that govt. agent SHALL be given the right to spy and conduct mass surveillance upon the citizens JUST AS search & seizures, if "reasonable" motive is deemed (by some private or unellected agents)

- **d)**  yes it does, see [Art. 14: Structure of the Government](https://github.com/liberland/Constitution/blob/Anniversary-8/Constitution.md#section-1-defining-principles) where the Judiciary is clearly defined as one of the branches of the govt.
However, going by your premise, it BOTH implies that the Constitution gives some private entity over the citizens of Liberland, as well as the fact that "the Judiciary" is not (exclusively) a govt. branch, yet it has governmental powers (given by this Constitution)

- **e)** mass surveillance + search & seizures **ARE criminal acts** when committed by private individuals and companies/organizations, and the current form of the Constitution just gave those exact private entities to enact them, against the exact same victims you mentioned above (+the rest of the population, not just women)

- **f)** are we regulating marriage now? Or just the religious ones?


### Already happening, IRL examples and their consequences:

A Liberlander on the official Telegram already declared himself as founder & head of the "_First Capitalist Party of Liberland_" right after the country's founding (also possibly supreme leader the "First Communist Party ~") self-admittedly only in his head, just by thinking about it, without any support (or knowledge) of the citizens or any official institution, without any documentation being posted or submitted. 
The same way a "private Judiciary" can be declared by such individuals and abuse the powers delegated (without any right) by the current Constitution, by your interpretation of it having a (semi?)private Judiciary.


### TL;DR: Please **REMOVE mass surveillance, search and seizures** from the Constitution **entirely.**

**Comment by michalptacnik** [comment]
a) Criminal investigation is a good motive. If we left it to NAP, any investigator would be placed in a game theory situation of playing a Schelling point-like game "guess if the investigated is guilty." The stakes: NAP implied damages for intruding on their property. This would paralyze any investigations where the victims are poor. It would, in turn, lead to a plutocracy of the darkest kind, where only the rich have justice while the poor live under the feudal powers of their "noble protectors."

b) Judiciary is a branch of the government, but functions of the Judiciary can be privatized, and private courts are a part of the Judiciary. 

c) The agent will be given that right when the judge allows it, primarily for criminal investigations. Why that is necessary, see under a).

d) If you, as a private citizen, voluntarily assert the jurisdiction to a private court, you will go there. If you appeal, you will ultimately end by the State Supreme Court or, if that is the hierarchy of the court you agree to, at State courts of appeal. That is if you appeal. If not, you will just be given justice by the community you feel most comfortable with according to the law you feel bound to. 

e) Not necessarily. In NAP theory, I would be justified to surveil you if you had broken NAP first. Our Constitution tries to regulate this. If they do it unprovoked, it would be a NAP infringement, possibly a crime if our criminal law criminalizes it.

f) we can't limit marriage. We can regulate it in the sense of having State marriages. They will not be monopoly State marriages, and you can call yourself married when you marry entirely privately. 

TL;DR: I need to understand how criminal investigation will happen in Liberland and "use core NAP rules" is not a good answer. We need to create a system where NAP rules are broken. Where for certain actions of the investigator, even if they cause damage, the investigator or the state are not liable. 

Otherwise, we can resolve this **by simple deletion of the entire "offending" paragraph".** The system would satisfy this requirement. But then, the right would not be specified in the Constitution and would be left for a Judge to research and assert in jurisprudence.

**Comment by michalptacnik** [comment]
Added freedom of information into the branch "no bill of rights"; inspired by the Swiss constitution, as agreed. Is this sufficient? (please see last change log in History)
